### PR TITLE
Fixing unhandled exception when closing a wav handle

### PR DESCRIPTION
**Pull request by luarvique**
This change should fix intermittent unhandled exception when closing a non-existant wav handle:

```
Nov 06 22:39:07 renegade openwebrx[434]: Traceback (most recent call last):
Nov 06 22:39:07 renegade openwebrx[434]:   File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
Nov 06 22:39:07 renegade openwebrx[434]:     self.run()
Nov 06 22:39:07 renegade openwebrx[434]:   File "/usr/lib/python3.10/threading.py", line 1378, in run
Nov 06 22:39:07 renegade openwebrx[434]:     self.function(*self.args, **self.kwargs)
Nov 06 22:39:07 renegade openwebrx[434]:   File "/usr/lib/python3/dist-packages/owrx/audio/wav.py", line 88, in switchFiles
Nov 06 22:39:07 renegade openwebrx[434]:     file.close()
Nov 06 22:39:07 renegade openwebrx[434]: AttributeError: 'NoneType' object has no attribute 'close'
Nov 06 22:39:15 renegade openwebrx[434]: Exception in thread Thread-3277:
Nov 06 22:39:15 renegade openwebrx[434]: Traceback (most recent call last):
Nov 06 22:39:15 renegade openwebrx[434]:   File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
Nov 06 22:39:15 renegade openwebrx[434]:     self.run()
Nov 06 22:39:15 renegade openwebrx[434]:   File "/usr/lib/python3.10/threading.py", line 1378, in run
Nov 06 22:39:15 renegade openwebrx[434]:     self.function(*self.args, **self.kwargs)
Nov 06 22:39:15 renegade openwebrx[434]:   File "/usr/lib/python3/dist-packages/owrx/audio/wav.py", line 88, in switchFiles
Nov 06 22:39:15 renegade openwebrx[434]:     file.close()
```

### CODE REVIEW

File: owrx/audio/wav.py [hunk 1]
```diff
@@ -85,7 +85,9 @@ def switchFiles(self):
             file = self.wavefile
             self.wavefile = self.getWaveFile()
 
-        file.close()
+        if file != None:
```
Comment (jketterl) [inline owrx/audio/wav.py:88]: The python style guide recommends checking for `None` values by means of the `is` operator: https://peps.python.org/pep-0008/#programming-recommendations

`if file is not None:`

**Review by jketterl — COMMENTED** [review]
Is there any information as to why the variable is `None`? I'd prefer to fix the actual condition that leads to this error over just avoiding it.

File: owrx/audio/wav.py (hunk 1 above)
Comment (luarvique) [reply owrx/audio/wav.py:88]: Not a problem, will make the change.

**Comment by luarvique** [comment]
Further development done at

https://github.com/luarvique/openwebrx

**Pull request closed without merging**